specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2022-10-01/costmanagement.benefits.json

```diff
@@ -0,0 +1,1002 @@
+        "name": "grain",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Hourly",
+            "description": "Hourly grain corresponds to value per hour.",
+            "name": "Hourly"
+          },
+          {
+            "value": "Daily",
+            "description": "Hourly grain corresponds to value per day.",
+            "name": "Daily"
+          },
+          {
+            "value": "Monthly",
+            "description": "Hourly grain corresponds to value per month.",
+            "name": "Monthly"
+          }
+        ]
+      }
+    },
+    "BenefitResource": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Resource name.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type.",
+          "readOnly": true
+        },
+        "kind": {
+          "description": "Reservation or SavingsPlan.",
+          "$ref": "#/definitions/BenefitPlanKind"
+        }
+      },
+      "description": "The Resource model definition.",
+      "x-ms-azure-resource": true
+    },
+    "BenefitUtilizationSummariesListResult": {
+      "description": "Result of listing benefit utilization summaries.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of benefit utilization summaries.",
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/BenefitUtilizationSummary"
+          }
+        },
+        "nextLink": {
+          "description": "The link (URL) to the next page of results.",
+          "type": "string",
+          "readOnly": true,
+          "format": "uri"
+        }
+      }
+    },
+    "BenefitUtilizationSummary": {
+      "description": "Benefit utilization summary resource.",
+      "discriminator": "kind",
+      "type": "object",
+      "required": [
+        "kind"
+      ],
+      "allOf": [
+        {
+          "$ref": "#/definitions/BenefitUtilizationResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "description": "Reservation, SavingsPlan or IncludedQuantity.",
+          "$ref": "#/definitions/BenefitUtilizationSummaryKind"
+        }
+      }
+    },
+    "BenefitUtilizationSummaryKind": {
+      "type": "string",
+      "description": "Kind of the benefit utilization summary.",
+      "enum": [
+        "IncludedQuantity",
+        "Reservation",
+        "SavingsPlan"
+      ],
+      "x-ms-enum": {
+        "name": "benefitUtilizationSummaryKind",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "IncludedQuantity",
+            "description": "Benefit utilization summary is for an IncludedQuantity resource like Databricks.",
```

Comment:
>ion summary is for an IncludedQuantity resource like Databricks.", [](http://example.com/codeflow?start=44&length=66)

Whats an 'IncludedQuantity' resource? Can this be turned into plainer English that won't require further research?

Reply:
This enum definition was duplicated. Hence removed.